web/src/search_suggestion.ts

```diff
@@ -635,6 +635,12 @@ function get_channels_filter_suggestions(last: NarrowTerm, terms: NarrowTerm[]):
     } else {
         description_html = "All public channels";
     }
+    let archived_description_html;
+    if (page_params.is_spectator || current_user.is_guest) {
+        archived_description_html = "All archived channels that you can view";
+    } else {
+        archived_description_html = "All archived channels";
+    }
```

Comment:
This check doesn't make sense; it should probably always be just "Archived channels".